src/main/java/com/exasol/glue/query/ExportQueryGenerator.java

```diff
@@ -0,0 +1,56 @@
+package com.exasol.glue.query;
+
+import com.exasol.glue.ExasolOptions;
+
+/**
+ * A class that generates an Exasol {@code EXPORT} query.
+ */
+public final class ExportQueryGenerator extends AbstractQueryGenerator {
+    private static final String EXPORT_QUERY_FOOTER = "WITH COLUMN NAMES";
+    private String filesPrefix;
+    private int numberOfFiles;
+
+    /**
+     * Creates a new instance of {@link ExportQueryGenerator}.
+     *
+     * @param options       user provided options
+     * @param filesPrefix   a prefix string that indicate folder inside a bucket
```

Comment:
I will keep it, since here we are only interested on Exasol query (not filesystem), `filesPrefix` expresses that we are creating `'filePrefix/part-0000.csv'` lines for the query.